modules/backend/formwidgets/Repeater.php

```diff
@@ -220,6 +220,21 @@ protected function processItems()
             ? post($this->formField->getName())
             : $this->getLoadValue();
 
+        // Detect when a child widget is trying to run an AJAX handler
+        // outside of the form element that contains all the repeater
+        // fields that would normally be used to identify that case
+        $handler = $this->controller->getAjaxHandler();
+        if (!$this->loaded && starts_with($handler, $this->alias . 'Form')) {
```

Comment:
@bennothommo the starts_with() should handle the child repeaters, although I haven't tested.

Reply:
@bennothommo tested it, it works perfectly in the child repeater too!

Reply:
Perhaps we can revisit the child repeater approach to see if we can simplify it.

Reply:
@LukeTowers True, but the regex below would only pick up one index, it wouldn't pick up the other indexes. Unless I'm misreading it.

Reply:
@LukeTowers looks good from first glances. Tested with nested Repeaters too. I guess now the question remains, what could this break in some cases? If nothing, I believe this is suitable

Reply:
@LukeTowers awesome. While I'm sure we can simplify it in places, I'm also hesitant to make too big a change to them after the last fiasco. :P

Reply:
@LukeTowers Since the index is passed on via the headers as the handler name and the Repeater simply extracts the index from it, without making any alterations, it makes it fool-proof in my opinion. I thought maybe if the Repeater had a number of RecordFinders, there could be problems but after testing I realized the key is just about stripping part of a string, so there should be no harm done in any case. My analytical 0.02